File: src/SOP8/Base/History/Base.History.IBLL/Request/RequestSOPHistory.cs
using Response.Request;

namespace Base.History.IBLL.Request
{
    public class RequestSOPHistory : RequestPage
    {
        private int m_nBeginYear = -1;
        private int m_nBeginMonth = -1;
        private int m_nBeginDay = -1;
        private int m_nEndYear = -1;
        private int m_nEndMonth = -1;
        private int m_nEndDay = -1;
        private int m_nSiteNo = -1;
        private string m_strDisasterCategoryName = null;
        private string m_strActionStepName = null;
        private string m_strUserName = null;

        public int BeginYear
        {
            get { return m_nBeginYear; }
            set { m_nBeginYear = value; }
        }

        public int BeginMonth
        {
            get { return m_nBeginMonth; }
            set { m_nBeginMonth = value; }
        }

        public int BeginDay
        {
            get { return m_nBeginDay; }
            set { m_nBeginDay = value; }
        }

        public int EndYear
        {
            get { return m_nEndYear; }
            set { m_nEndYear = value; }
        }

        public int EndMonth
        {
            get { return m_nEndMonth; }
            set { m_nEndMonth = value; }
        }

        public int EndDay
        {
            get { return m_nEndDay; }
            set { m_nEndDay = value; }
        }

        public int SiteNo
        {
            get { return m_nSiteNo; }
            set { m_nSiteNo = value; }
        }

        // null이면 전체
        public string DisasterCategoryName
        {
            get { return m_strDisasterCategoryName; }
            set { m_strDisasterCategoryName = value; }
        }

        // null이면 전체
        public string ActionStepName
        {
            get { return m_strActionStepName; }
            set { m_strActionStepName = value; }
        }

        // null이면 전체
        public string UserName
        {
            get { return m_strUserName; }
            set { m_strUserName = value; }
        }
    }
}

File: src/SOP8/Egovframework/Base/base-history-ibll/src/main/java/egovframework/base/history/ibll/request/RequestSOPHistory.java
package egovframework.base.history.ibll.request;

import egovframework.base.response.request.RequestPage;

public class RequestSOPHistory extends RequestPage {

    private int beginYear = -1;
    private int beginMonth = -1;
    private int beginDay = -1;
    private int endYear = -1;
    private int endMonth = -1;
    private int endDay = -1;
    private int siteNo = -1;

    private String disasterCategoryName = null; // null이면 전체
    private String actionStepName = null;       // null이면 전체
    private String userName = null;             // null이면 전체

    // Getters and Setters

    public int getBeginYear() {
        return beginYear;
    }

    public void setBeginYear(int beginYear) {
        this.beginYear = beginYear;
    }

    public int getBeginMonth() {
        return beginMonth;
    }

    public void setBeginMonth(int beginMonth) {
        this.beginMonth = beginMonth;
    }

    public int getBeginDay() {
        return beginDay;
    }

    public void setBeginDay(int beginDay) {
        this.beginDay = beginDay;
    }

    public int getEndYear() {
        return endYear;
    }

    public void setEndYear(int endYear) {
        this.endYear = endYear;
    }

    public int getEndMonth() {
        return endMonth;
    }

    public void setEndMonth(int endMonth) {
        this.endMonth = endMonth;
    }

    public int getEndDay() {
        return endDay;
    }

    public void setEndDay(int endDay) {
        this.endDay = endDay;
    }

    public int getSiteNo() {
        return siteNo;
    }

    public void setSiteNo(int siteNo) {
        this.siteNo = siteNo;
    }

    public String getDisasterCategoryName() {
        return disasterCategoryName;
    }

    public void setDisasterCategoryName(String disasterCategoryName) {
        this.disasterCategoryName = disasterCategoryName;
    }

    public String getActionStepName() {
        return actionStepName;
    }

    public void setActionStepName(String actionStepName) {
        this.actionStepName = actionStepName;
    }

    public String getUserName() {
        return userName;
    }

    public void setUserName(String userName) {
        this.userName = userName;
    }
}